components/blitz/resources/omero/Constants.ice

```diff
@@ -214,14 +214,6 @@ module omero {
      **/
     module analysis {
 
-       /** namespaces related to the FLIM analysis. **/
```

Comment:
The problem is that the scripts have already been deleted
The URL available is no longer working.
So in theory, I agree, in practise it is already totally broken